pkg/cloudprovider/vsphere/loadbalancer/cleanup.go

```diff
@@ -88,18 +89,18 @@ func (p *lbProvider) doCleanupStep(clusterName string, client clientcorev1.Servi
 		}
 	}
 
-	return p.CleanupServices(clusterName, services)
+	return p.CleanupServices(clusterName, services, false)
 }
 
-func (p *lbProvider) CleanupServices(clusterName string, validServices map[types.NamespacedName]corev1.Service) error {
+func (p *lbProvider) CleanupServices(clusterName string, validServices map[types.NamespacedName]corev1.Service, ensureLBServiceDeleted bool) error {
 	ipPoolIds := sets.NewString()
 	for _, name := range p.classes.GetClassNames() {
 		class := p.classes.GetClass(name)
 		ipPoolIds.Insert(class.ipPool.Identifier)
 	}
 
 	lbs := map[types.NamespacedName]struct{}{}
-	servers, err := p.access.ListVirtualServers(ClusterName)
```

Comment:
What was `ClusterName` pointing to previously?

Reply:
`ClusterName` is the global variable which is set by by the `--cluster-name` flag.
(see https://github.com/kubernetes/cloud-provider-vsphere/blob/44a75efaef85b259f75a2cf729b60149dbb9353f/cmd/vsphere-cloud-controller-manager/main.go#L93-L95)
This change makes only a difference, if the function CleanupServices is used externally. Otherwise the passed `clusterName` is the same as before
(see https://github.com/kubernetes/cloud-provider-vsphere/blob/master/pkg/cloudprovider/vsphere/cloud.go#L118)